Added argument for choosing focus method

Tscunami opened the pull request:
Added argument for choosing focus method.
Closing #1

Slavaqq reviewed (changes requested):
- Mention of this change should be add to the README.md.
 - Why did not you add the setup of focus_method to the parent class?

Tscunami commented:
Thank you for your feedback. In my new commit 2212677 I added a section about this feature to README.md, and I also moved setup of focus_method to the parent class.

Slavaqq commented on urpaform/elements.py:
```diff
@@ -9,7 +9,15 @@
     """A private class representing a common element in a form."""
 
-    def __init__(self, element: urpa.AppElement, show_in_log: bool = True, allow_check: bool = True) -> None:
+    _FOCUS_ACTION_IS_IN = ("Default", "Mouse", "None")
```
Other string options don´t start with the capital letters. We should unify this.

Tscunami replied:
Fixed.

Slavaqq commented on README.md:
```diff
@@ -73,6 +73,28 @@ test_form.add(combo_field, "Saturday")
+This attribute can be specified for all types of field.
+There are 3 options:   
+- `Default` - set up by default, can be used with a hidden desktop, but takes longer time to process  
+- `Mouse` - the field is enabled with a mouse click  
```
we should use the established terminology word `focus`

Tscunami replied:
Fixed.

Tscunami commented on README.md:
````diff
@@ -73,6 +73,28 @@ test_form.add(combo_field, "Saturday")
+- `None` - the field will not be enabled   
+
+```python
+from urpaform import EditElement
````
Fixed

Tscunami commented:
In my last commit c9c0ff3, I resolved all issues above.
